gemfiles/rails6.1.gemfile.lock

```diff
@@ -0,0 +1,267 @@
+PATH
+  remote: ..
+  specs:
+    factory_bot_rails (6.2.0)
+      factory_bot (~> 6.2.0)
+      railties (>= 5.0.0)
+
+GEM
+  remote: https://rubygems.org/
+  specs:
+    actioncable (6.1.4.1)
+      actionpack (= 6.1.4.1)
+      activesupport (= 6.1.4.1)
+      nio4r (~> 2.0)
+      websocket-driver (>= 0.6.1)
```

Comment:
probably don't want to commit .lock in, and instead add `gemfiles/*lock` to the `.gitignore` file